Retrieved from https://github.com/elogind/elogind/issues/258

From: Mark Hindley <mark@hindley.org.uk>
Date: Wed, 24 May 2023 10:39:41 +0100
Subject: Fixup_executable_rpath

./meson.build sets

 install_rpath : rootlibexecdir

however src/shared/meson.build sets

libshared = shared_library(
          [snip]
        install_dir : rootpkglibdir
        )
---
 meson.build | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/meson.build b/meson.build
index 694a2fd..a575f69 100644
--- a/meson.build
+++ b/meson.build
@@ -2903,7 +2903,7 @@ executable('elogind',
                 dependencies : [threads,
                                 libacl,
                                 libudev],
-                install_rpath : rootlibexecdir,
+                install_rpath : rootpkglibdir,
                 install : true,
                 install_dir : rootlibexecdir)
 
@@ -2913,7 +2913,7 @@ exe = executable('loginctl',
                         link_with : [libshared],
                         dependencies : [threads,
                                         libudev],
-                        install_rpath : rootlibexecdir,
+                        install_rpath : rootpkglibdir,
                         install : true,
                         install_dir : rootbindir)
 public_programs += [exe]
@@ -2923,7 +2923,7 @@ exe = executable('elogind-inhibit',
                         include_directories : includes,
                         link_with : [libshared],
                         dependencies : [threads],
-                        install_rpath : rootlibexecdir,
+                        install_rpath : rootpkglibdir,
                         install : true,
                         install_dir : rootbindir)
 public_programs += [exe]
@@ -4283,7 +4283,7 @@ executable('elogind-uaccess-command',
                         libshared],
            dependencies: [libacl,
                           libudev],
-           install_rpath : rootlibexecdir,
+           install_rpath : rootpkglibdir,
            install : true,
            install_dir : rootlibexecdir)
 #endif // 0
